dashboard/state_aggregator.py

```diff
@@ -443,6 +445,7 @@ async def list_objects(self, *, option: ListApiOptions) -> ListApiResponse:
 
         result = []
         memory_table = memory_utils.construct_memory_table(worker_stats)
+        callsite_enabled = True
```

Comment:
Hmm I did it in case someone doesn't start ray with `RAY_record_ref_creation_sites`, but set this when driver is created. (In this case, callsite is recorded only for driver case). But maybe it is okay to just ignore this case.. let me fix it